genesis/parsing/accountsParser_test.go

```diff
@@ -92,47 +96,46 @@ func createDelegatedInitialAccount(address string, delegatedBytes []byte, delega
 	return ia
 }
 
-func TestNewAccountsParser_NilEntireBalanceShouldErr(t *testing.T) {
+func TestNewAccountsParser_NilInitialAccountsShouldErr(t *testing.T) {
 	t.Parallel()
 
 	args := createMockAccountsParserArgs()
-	args.EntireSupply = nil
+	args.InitialAccounts = nil
 
 	ap, err := parsing.NewAccountsParser(args)
 
 	assert.True(t, check.IfNil(ap))
-	assert.True(t, errors.Is(err, genesis.ErrNilEntireSupply))
+	assert.True(t, errors.Is(err, errorsMx.ErrNilInitialAccounts))
 }
 
-func TestNewAccountsParser_ZeroEntireBalanceShouldErr(t *testing.T) {
+func TestNewAccountsParser_NilEntireBalanceShouldErr(t *testing.T) {
 	t.Parallel()
 
 	args := createMockAccountsParserArgs()
-	args.EntireSupply = big.NewInt(0)
+	args.EntireSupply = nil
 
 	ap, err := parsing.NewAccountsParser(args)
 
 	assert.True(t, check.IfNil(ap))
-	assert.True(t, errors.Is(err, genesis.ErrInvalidEntireSupply))
+	assert.True(t, errors.Is(err, genesis.ErrNilEntireSupply))
 }
 
-func TestNewAccountsParser_BadFilenameShouldErr(t *testing.T) {
```

Comment:
Is this being tested somewhere else now?

Reply:
not really, the loading of file happens in `nodeRunner`